spec/services/flowschedule_rota_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'spec_helper'
+require_relative '../../services/floatschedule_rota'
+
+describe FloatscheduleRota do
+  let(:spec_dir)         { File.dirname(__FILE__)                                            }
+  let(:example_response) { File.read(spec_dir + '/../fixtures/float_tasks_api_response.json') }
```

Comment:
I wouldn't bother lining these braces up myself, but *almost* lining them up is even worse!